modules/core/remove_orphaned.py

```diff
@@ -89,14 +112,31 @@ def rem_orphaned(self):
             }
             self.config.send_notifications(attr)
             # Delete empty directories after moving orphan files
-            logger.info("Cleaning up any empty directories...")
             if not self.config.dry_run:
-                for file in orphaned_files:
-                    src = file.replace(self.root_dir, self.remote_dir)
-                    dest = os.path.join(self.orphaned_dir, file.replace(self.root_dir, ""))
-                    util.move_files(src, dest, True)
-                    orphaned_parent_path.add(os.path.dirname(file).replace(self.root_dir, self.remote_dir))
-                    for parent_path in orphaned_parent_path:
-                        util.remove_empty_directories(parent_path, "**/*")
+                orphaned_parent_path = set(self.pool.map(move_orphan, orphaned_files))
+                logger.print_line("Removing newly empty directories", self.config.loglevel)
+                self.pool.starmap(util.remove_empty_directories, zip(orphaned_parent_path, repeat("**/*")))
+
         else:
             logger.print_line("No Orphaned Files found.", self.config.loglevel)
+
+    def cleanup_pool(self):
+        self.pool.close()
+        self.pool.join()
+
+
+def get_full_path_of_torrent_files(torrent_files, save_path):
+    fullpath_torrent_files = []
+    for file in torrent_files:
+        fullpath = os.path.join(save_path, file)
+        # Replace fullpath with \\ if qbm is running in docker (linux) but qbt is on windows
+        fullpath = fullpath.replace(r"/", "\\") if ":\\" in fullpath else fullpath
+        fullpath_torrent_files.append(fullpath)
+    return fullpath_torrent_files
+
+
+def move_orphan(file):
+    src = file.replace(_config.root_dir, _config.remote_dir)  # Could be optimized to only run when root != remote
+    dest = os.path.join(_config.orphaned_dir, file.replace(_config.root_dir, ""))
+    util.move_files(src, dest, True)
+    return os.path.dirname(file).replace(_config.root_dir, _config.remote_dir)  # Another candidate for micro optimizing
```

Comment:
```suggestion
    return os.path.dirname(file).replace(_config.root_dir, _config.remote_dir)  # Another candidate for micro optimizing

```
Missing newline in EOF.